ide/plugins/java-format/src/content/formats/webdriver-testng.js

```diff
@@ -0,0 +1,523 @@
+  if (this.e1.toString().match(/^\d+$/)) {
+    // int
+    return this.e1.toString() + " == " + this.e2.toString();
+  } else {
+    // string
+    return this.e1.toString() + ".equals(" + this.e2.toString() + ")";
+  }
+};
+
+Equals.prototype.assert = function() {
+  return "assertEquals(" + this.e1.toString() + ", " + this.e2.toString() + ");";
+};
+
+Equals.prototype.verify = function() {
+  return verify(this.assert());
+};
+
+NotEquals.prototype.toString = function() {
+  return "!" + this.e1.toString() + ".equals(" + this.e2.toString() + ")";
+};
+
+NotEquals.prototype.assert = function() {
+  return "assertThat(" + this.e1.toString() + ", is(not(" + this.e2.toString() + ")));";
+};
+
+NotEquals.prototype.verify = function() {
+  return verify(this.assert());
+};
+
+function joinExpression(expression) {
+  return "join(" + expression.toString() + ", ',')";
+}
+
+function statement(expression) {
+  var s = expression.toString();
+  if (s.length == 0) {
+    return null;
+  }
+  return s + ';';
+}
+
+function assignToVariable(type, variable, expression) {
+  return type + " " + variable + " = " + expression.toString();
+}
+
+function ifCondition(expression, callback) {
+  return "if (" + expression.toString() + ") {\n" + callback() + "}";
+}
+
+function assertTrue(expression) {
+  return "assertTrue(" + expression.toString() + ");";
+}
+
+function assertFalse(expression) {
+  return "assertFalse(" + expression.toString() + ");";
+}
+
+function verify(statement) {
+  return "try {\n" +
+      indents(1) + statement + "\n" +
+      "} catch (Error e) {\n" +
+      indents(1) + "verificationErrors.append(e.toString());\n" +
+      "}";
+}
+
+function verifyTrue(expression) {
+  return verify(assertTrue(expression));
+}
+
+function verifyFalse(expression) {
+  return verify(assertFalse(expression));
+}
+
+RegexpMatch.prototype.toString = function() {
+  if (this.pattern.match(/^\^/) && this.pattern.match(/\$$/)) {
+    return this.expression + ".matches(" + string(this.pattern) + ")";
+  } else {
+    return "Pattern.compile(" + string(this.pattern) + ").matcher(" + this.expression + ").find()";
+  }
+};
+
+function waitFor(expression) {
+  return "for (int second = 0;; second++) {\n" +
+      "\tif (second >= 60) fail(\"timeout\");\n" +
+      "\ttry { " + (expression.setup ? expression.setup() + " " : "") +
+      "if (" + expression.toString() + ") break; } catch (Exception e) {}\n" +
+      "\tThread.sleep(1000);\n" +
+      "}\n";
+}
+
+function assertOrVerifyFailure(line, isAssert) {
+  var message = '"expected failure"';
+  var failStatement = "fail(" + message + ");";
+  return "try { " + line + " " + failStatement + " } catch (Throwable e) {}";
+}
+
+function pause(milliseconds) {
+  return "Thread.sleep(" + parseInt(milliseconds, 10) + ");";
+}
+
+function echo(message) {
+  return "System.out.println(" + xlateArgument(message) + ");";
+}
+
+function formatComment(comment) {
+  return comment.comment.replace(/.+/mg, function(str) {
```

Comment:
Fixed in [#7e0de8f](https://github.com/rasika90/selenium/commit/7e0de8f7253c45ace8d0fd539f4712e1f91349c1).